crates/fluvio-benchmark/src/benchmark_config/mod.rs

```diff
@@ -58,13 +58,10 @@ pub struct BenchmarkConfig {
     pub producer_linger: Duration,
```

Comment:
Would prefer separating configuration into it's own domain:
```
     ProducerBenchmarkConfig
```

Reply:
I agree. The reason it's not already that way is that it was pretty complicated to avoid a massive nested for-loop with generating the cross-product of the dimensions in `BenchmarkMatrix::cross_iterate()`. It's not obvious yet how I would be able to accomplish this change easily without some other big tradeoffs.